ui/server.py

```diff
@@ -407,6 +408,22 @@ def update_discovery_group():
         return 'OK', 200
 
 
+@app.route('/update_similar_discoveries', methods=['POST'])
+def update_similar_discoveries():
+    target_snippet = request.form.get('snippet')
+    state = request.form.get('state')
+    url = request.form.get('url')
+    file = request.form.get('file')
+
+    response1 = c.update_discovery_group(state, url, file, target_snippet)
+    response2 = c.update_similar_snippets(target_snippet,
+                                          state,
+                                          url)
+    if response1 is False or response2 is False:
```

Comment:
`if not response1 or not response2:`
(more clear)

Reply:
I agree, but in the update_discovery_group, there `if response is False`